components/suggest/src/query.rs

```diff
@@ -297,4 +321,25 @@ mod test {
         // characters).
         assert!(FtsQuery::new("run").match_required_stemming("running shoes"));
     }
+
+    #[test]
+    fn test_full_keywords_to_fts_content() {
+        check_full_keywords_to_fts_content(["a", "b", "c"], "a b c");
+        check_full_keywords_to_fts_content(["a", "b c"], "a b c");
+        check_full_keywords_to_fts_content(["a", "b c a"], "a b c");
+        check_full_keywords_to_fts_content(["a", "b C A"], "a b c");
+    }
+
+    fn check_full_keywords_to_fts_content<const N: usize>(input: [&str; N], expected: &str) {
+        let mut expected_counts = HashMap::<&str, usize>::new();
+        let mut actual_counts = HashMap::<&str, usize>::new();
+        for term in expected.split_whitespace() {
+            *expected_counts.entry(term).or_default() += 1;
+        }
+        let fts_content = full_keywords_to_fts_content(input);
+        for term in fts_content.split_whitespace() {
+            *actual_counts.entry(term).or_default() += 1;
+        }
+        assert_eq!(actual_counts, expected_counts);
```

Comment:
Why not simply `assert_eq!(fts_content, expected)`? That would simplify this test logic, making bugs less likely, and I think it's OK to make an assumption about the order of words in `fts_content`.

Reply:
I see, I didn't realize that.